src/uct/cuda/cuda_ipc/cuda_ipc_ep.c

```diff
@@ -2,14 +2,25 @@
  * Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
  * See file LICENSE for terms.
  */
+/**
+ *  Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+ *  See COPYRIGHT for license information
+ */
```

Comment:
multiple copyrights should go to same comments.  see https://github.com/openucx/ucx/blob/master/src/uct/ugni/rdma/ugni_rdma_ep.c#L2